README.md

```diff
@@ -99,1198 +36,30 @@ To test if the installation is successful, you may execute
-     "shuffle_poscar":   false,
-     "fp_task_max":  100,
-     "fp_task_min":  10,
-     "fp_pp_path":   ".",
-     "fp_pp_files":  [],
-     "external_input_path": "./cp2k_make_fp_files/exinput/template.inp",
-     "_comment":     " that's all 
-     }
-```
-
-the following essential section should be provided in user template
-
-```
-
- &FORCE_EVAL
-   # add this line if you need to fit virial
-   STRESS_TENSOR ANALYTICAL
-   &PRINT
-     &FORCES ON
-     &END FORCES
-     # add this line if you need to fit virial
-     &STRESS_TENSOR ON
-     &END FORCES
-   &END PRINT
-   &SUBSYS
-     &CELL
-       ABC LEFT FOR DPGEN
-     &END CELL
-     &COORD
-     @include coord.xyz
-     &END COORD
-   &END SUBSYS
-&END FORCE_EVAL
-```
+* [`dpgen run`](https://docs.deepmodeling.com/projects/dpgen/en/latest/run/): Main process of Deep Generator.
+* [Init](https://docs.deepmodeling.com/projects/dpgen/en/latest/init/): Generating initial data.
+  * `dpgen init_bulk`: Generating initial data for bulk systems.
+  * `dpgen init_surf`: Generating initial data for surface systems.
+  * `dpgen init_reaction`: Generating initial data for reactive systems.
+* [`dpgen simplify`](https://docs.deepmodeling.com/projects/dpgen/en/latest/simplify/): Reducing the amount of existing dataset.
+* [`dpgen autotest`](https://docs.deepmodeling.com/projects/dpgen/en/latest/autotest/): Autotest for Deep Potential.
 
-## Autotest: Autotest for Deep Generator
-Suppose that we have a potential (can be DFT, DP, MEAM ...), `autotest` helps us automatically calculate M properties on N configurations. The folder where the `autotest` runs is called the working directory of `autotest`. Different potentials should be tested in different working directories.
+For detailed usage and parameters, read [DP-GEN documentation](https://docs.deepmodeling.com/projects/dpgen/).
 
-A property is tested in three steps: `make`, `run` and `post`. `make` prepares all computational tasks that are needed to calculate the property. For example to calculate EOS, `make` prepares a series of tasks, each of which has a scaled configuration with certain volume, and all necessary input files necessary for starting a VASP, ABACUS, or LAMMPS calculations. `run` sends all the computational tasks to remote computational resources defined in a machine configuration file like `machine.json`, and automatically collects the results when remote calculations finish. `post` calculates the desired property from the collected results.
+## Tutorials and examples
 
-### Relaxation
+* [Tutorials](https://tutorials.deepmodeling.com/en/latest/Tutorials/DP-GEN/): basic tutorials for DP-GEN.
+* [Examples](examples): input files in [JSON](https://docs.python.org/3/library/json.html) format.
+* [Publications](https://deepmodeling.com/blog/papers/dpgen/): Published research articles using DP-GEN.
+* [User guide](https://docs.deepmodeling.com/projects/dpgen/en/latest/user-guide/): frequently asked questions listed in troubleshooting.
```

Comment:
this one is broken. I didn't check every link, could you please do so?

Reply:
These links will be available once the devel branch is merged to the master branch. At present the link can be visited by replacing `latest` with `devel`.